managedtenants/cli/__init__.py

```diff
@@ -41,6 +41,7 @@ def __init__(self):
         parser.add_argument(
             "--addons-dir",
             type=self._validate_addons_dir,
+            default="addons",
```

Comment:
Can we make this flag as required instead of having a default? 

I'm not sure how this default value would help. Earlier when the CLI used to live in the managed-tenants repo, we had the complete path to the addons directory as the default, and that made sense because both, the CLI and the addons directory were in the same repo.

Now that the CLI has been separated, I'm not sure if it makes sense to have a default because the addons directory (or the managed-tenats repo) could be anywhere on the system (local or CI) running the CLI. Currently, we pass the complete path to the addons dir in the managed-tenants CI - https://gitlab.cee.redhat.com/service/managed-tenants/-/blob/main/Makefile#L51


WDYT about making this flag as required, and throwing an error when:
- No value has been provided for this flag, or
- value has been provided, but the directory is invalid

Reply:
I love this :heart: . Pushing the change.